capacity4more/modules/c4m/features_og/c4m_features_og_events/c4m_features_og_events.module

```diff
@@ -190,3 +199,55 @@ function c4m_features_og_events_facet_items_alter(&$build, &$settings) {
     }
   }
 }
+
+/**
+ * Implements hook_entity_info_alter().
+ *
+ * Add "Upcoming event" view mode.
+ */
+function c4m_features_og_events_entity_info_alter(&$entity_info) {
+  $entity_info['node']['view modes']['upcoming_event'] = array(
+    'label' => t('Upcoming event'),
+    'custom settings' => TRUE,
+  );
+}
+
+/**
+ * Fetches upcoming events (Events in the future that haven't started yet),
+ * Depending on the user's role (guest, logged-in),
+ * This function fetches the events from user's groups or public groups.
+ * 1 Event for anonymous user, 3 for logged in.
+ *
+ * @return string
+ *  HTML of rendered upcoming events.
+ */
+function c4m_features_og_events_upcoming_events() {
+  global $user;
+  $date = date('Y-m-d H:i:s', strtotime('+1 day'));
```

Comment:
This should be date('Y-m-d 00:00:00(, strtotime('+1 day'));

It should be any event starting on the next day.
Not any event starting later then now() + 1day.

As it implemented now:
* If it is now 14:00
* And the event starts tomorrow at 13:00
* The event will not be shown > **NOK**

See (screenshot taken on 11 February at 14:30):
The reason the item is not shown is because the calculated date-time does not keep track of the timezone.

![c4m-upcoming event](https://cloud.githubusercontent.com/assets/133124/6147860/f5474634-b1fb-11e4-87d4-6c942853e81a.png)